clang/test/Analysis/Checkers/WebKit/uncounted-obj-arg.cpp

```diff
@@ -368,6 +368,11 @@ class RefCounted {
   }
   RefPtr<RefCounted> trivial66() { return children[0]; }
   Ref<RefCounted> trivial67() { return *children[0]; }
+  struct point {
+    double x;
+    double y;
+  };
+  void trivial68() { point pt = { 1.0 }; }
```

Comment:
Not sure how much this matters for the test, but doesn't this leave `y` uninitialized?

Reply:
It implicitly initializes y. That's what this bug is about. I wasn't recognizing that type of implicit initialization.